### ISSUE TITLE
Make it work with wiredep: angular-meditor was not injected in your file.

### ISSUE THREAD

**Issue by osvathrobi** [ghinda/angular-meditor#11]
Using grunt-bower-install: 0.7.0 it is unable to add it to the index.html automatically.

Message is:
```
angular-meditor was not injected in your file.

Please go take a look in "app/bower_components/angular-meditor" for the file you need, then manually include it in your file.
```

Will post solution if I find it.

**Comment by osvathrobi** [comment]
It would help if you would could specify in the bower.json of your component the main files that need to be injected with dependency wiring tools.

Adding this to bower.json would solve the issue with grunt-wiredep and grunt-bower-install:
"main": ["dist/meditor.css", "dist/meditor.js"],

**Comment by ghinda** [comment]
Fixed and released as `1.4.2`. Thanks for reporting buddy!

![j45f9](https://cloud.githubusercontent.com/assets/325171/10304833/3df9362e-6c25-11e5-9195-974a23961217.gif)
